src/tir/transforms/plan_update_buffer_allocation_location.cc

```diff
@@ -111,6 +112,12 @@ class BufferAllocationLocator : public StmtExprMutator {
     collector(func->body);
     unmanaged_allocations_ = collector.unmanaged_allocations;
 
+    for (Var param : func->params) {
```

Comment:
Need test case for this

Reply:
I'd say that this PR's unit test already acts as a test case, because this change was needed to make that test pass.

Reply:
Update: This has now been merged via https://github.com/apache/tvm/pull/13906.